### ISSUE TITLE
[Feature]: Adding Monero URl support

### ISSUE THREAD

**Issue by XMRZombie** [simplex-chat/simplex-chat#5157]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

all

### Feature

Adding Monero URl support could help integrating Monero payments on a "easy" level, the system will ask for an installed to open the URl and everything done after is managed in-wallets.
[Monero URl formatting](https://github.com/monero-project/monero/wiki/URI-Formatting)

**Comment by GTO683** [comment]
**Against this feature** because this is a messenger, not an all in one life app. I believe implementing an unrelated project that increases attack surface and is outside of SimpleX devs hands isn’t a good idea. Especially crypto. Money is involved so this could be an ulterior motive. Wallets already exist.

**Comment by XMRZombie** [comment]
Yes. Because this isn't an all in one life app I believe this PR wont integrate a wallet but only the URl formatting.
And this wont increase SimpleX surface attack because it haves nothing to do with SimpleX, you're mixing things and the Monero community haves many research groups to carry it's own security.

**Comment by GTO683** [comment]
> Yes. Because this isn't an all in one life app I believe this PR wont integrate a wallet but only the URl formatting. And this wont increase SimpleX surface attack because it haves nothing to do with SimpleX, you're mixing things and the Monero community haves many research groups to carry it's own security.

It’s irrelevant if it’s a URI or not, the Monero team, a very specific cryptocurrency, is completely different from the SimpleX team, a private messenger. If something happens to Monero, it’ll reflect SimpleX. I don’t believe any cryptocurrencies should be endorsed nor encouraged to be used in association with SimpleX and I believe the SimpleX team’s efforts should probably be entirely on SimpleX and messaging as for the near future.

**Comment by XMRZombie** [comment]
> the Monero team, of a very specific cryptocurrency, is completely different from the SimpleX team, of a private messenger
Ok so simplex supports simple URls opening your regular browser, does simplex team is related to mozilla or google ? It's not. So your cons are full of nonsense

**Comment by GTO683** [comment]
> > the Monero team, of a very specific cryptocurrency, is completely different from the SimpleX team, of a private messenger
> 
> Ok so simplex supports simple URls opening your regular browser, does simplex team is related to mozilla or google ? It's not. So your cons are full of nonsense

A browser is a blank canvas, and there are unlimited browsers. Not just Chrome or Firefox. You are advocating for support of a SPECIFIC cryptocurrency and literally have the username “XMR Zombie” because you know this acts as an XMR co-sign / endorsement, which benefits you and your bags. The team has already said they’re trying to stay away from crypto as much as possible. Their words, not mine.

**Comment by XMRZombie** [comment]
> You act as a co-sign/endorsement of XMR

Yes I like and use monero for making privacy respecting transactions online. And i'd like to have integration of those urls in simplex client since my browser does. I advocated for a specific currency because it's in my current need but implementing URl formatting of other currencies (and protocols unrelated to blockchain/cryptocurrencies) should be done to enhance the client UX
